js/bitforex.js

```diff
@@ -0,0 +1,605 @@
+        let lastTradeTimestamp = this.safeFloat2 (order, 'lastTime');
+        let symbol = market['symbol'];
+        let sideId = this.safeInteger (order, 'tradeType');
+        let side = this.parseSide (sideId);
+        let type = undefined;
+        let price = this.safeFloat (order, 'orderPrice');
+        let average = this.safeFloat (order, 'avgPrice');
+        let amount = this.safeFloat (order, 'orderAmount');
+        let filled = this.safeFloat (order, 'dealAmount');
+        let remaining = amount - filled;
+        let statusId = this.safeInteger (order, 'orderState');
+        let status = this.parseOrderStatus (statusId);
+        let cost = filled * price;
+        let fee = this.safeFloat (order, 'tradeFee');
+        let result = {
+            'info': order,
+            'id': id,
+            'timestamp': timestamp,
+            'datetime': iso8601,
+            'lastTradeTimestamp': lastTradeTimestamp,
+            'symbol': symbol,
+            'type': type,
+            'side': side,
+            'price': price,
+            'cost': cost,
+            'average': average,
+            'amount': amount,
+            'filled': filled,
+            'remaining': remaining,
+            'status': status,
+            'fee': fee,
+        };
+        return result;
+    }
+
+    async fetchOrder (id, symbol = undefined, params = {}) {
+        await this.loadMarkets ();
+        let market = this.market (symbol);
+        let request = {
+            'symbol': this.marketId (symbol),
+            'orderId': id,
+        };
+        let response = await this.privatePostApiV1TradeOrderInfo (this.extend (request, params));
+        let order = this.parseOrder (response['data'], market);
+        return order;
+    }
+
+    async fetchOpenOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        let market = this.market (symbol);
+        let request = {
+            'symbol': this.marketId (symbol),
+            'state': 0,
+        };
+        let response = await this.privatePostApiV1TradeOrderInfos (this.extend (request, params));
+        return this.parseOrders (response['data'], market, since, limit);
+    }
+
+    async fetchOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        let market = this.market (symbol);
+        let requestOpen = {
+            'symbol': this.marketId (symbol),
+            'state': 0,
+        };
+        let responseOpen = await this.privatePostApiV1TradeOrderInfos (this.extend (requestOpen, params));
+        let openData = [];
+        if ('data' in responseOpen) {
+            openData = responseOpen['data'];
+        }
+        let requestClosed = {
+            'symbol': this.marketId (symbol),
+            'state': 1,
+        };
+        let responseClosed = await this.privatePostApiV1TradeOrderInfos (this.extend (requestClosed, params));
```

Comment:
Here we are making two requests. One for open orders and one for closed orders. 
API Endpoint: [`api/v1/trade/orderInfos`](https://github.com/bitforexapi/API_Docs/wiki/%E6%9F%A5%E8%AF%A2%E7%94%A8%E6%88%B7%E8%AE%A2%E5%8D%95%E4%BF%A1%E6%81%AF). The `state` parameter is required and there is no wildcard as far as I'm aware.